Components/typography/typography.css

```diff
@@ -0,0 +1,65 @@
+.h1 {
```

Comment:
you are using margin:1rem  in all the heading class differently
so you can group all of  them into one such as:-

.h1{
margin:1rem;
}

.h2{
margin:1rem
}

.h3{
margin:1rem
}

this can also be written as 
.h1, .h2, .h3{
margin:1rem;
}

Reply:
okay

Reply:
done